x11-libs/libxkbcommon/libxkbcommon-1.3.0.ebuild

```diff
@@ -45,7 +50,7 @@ multilib_src_configure() {
 	local emesonargs=(
 		-Ddefault_library="$(usex static-libs both shared)"
 		-Dxkb-config-root="${EPREFIX}/usr/share/X11/xkb"
-		-Denable-wayland=false # Demo applications
+		$(meson_use wayland enable-wayland)
```

Comment:
This controls the building and installation of
````
./usr/libexec/xkbcommon/xkbcli-interactive-wayland
./usr/share/man/man1/xkbcli-interactive-wayland.1.bz2
````

I see that we're already installing

````
./usr/libexec/xkbcommon/xkbcli-interactive-x11
./usr/share/man/man1/xkbcli-interactive-x11.1.bz2
````

with USE=X. (It also controls some other things)

So... seems fine. Is this tool useful for you? I'm just unsure of the rationale.

Reply:
Yeah, the USE=X requires to have `xorg-server` installed, which isn't required for wayland usage in gentoo and I'm not aware of another wayland tool similar to this one or x11-apps/xev (`WAYLAND_DEBUG=1` is nice but it doesn't decodes events in something a human could directly use).